pull_into_place/pipeline.py

```diff
@@ -59,6 +59,11 @@ def root_dir(self):
         return os.path.normpath(
                 os.path.join(self._root_dirname, self._root_basename))
 
+    def rosetta_inputs_dir(self, default=True):
```

Comment:
Would rather call this `config_dir`, I think.  The `rosetta_` prefix makes me think that this is part of a rosetta checkout.